src/normalize_attributes.cpp

```diff
@@ -117,11 +117,32 @@ auto tune_attribute(const std::vector<int64_t>& vec,
     return result;
 }
 
+auto tune_attribute(const value& val)
+{
+
+    std::vector<size_t> vec_attrs = val.to_vector<size_t>();
+    std::vector<size_t> result(vec_attrs.begin(), vec_attrs.end());
+    std::copy(vec_attrs.begin(), vec_attrs.end(), std::back_inserter(result));
+
+    return result;
+}
+
 bool normalize_attributes(operation& op, const std::vector<std::size_t>& lens)
 {
     bool tuned = false;
     auto attrs = op.attributes();
     auto val   = op.to_value();
+    if(attrs.contains("normalize_padding"))
+    {
+        if(val.at("padding").to_vector<size_t>().size() == 2 * (lens.size() - 2))
+            return true;
+        if(val.at("padding").to_vector<size_t>().size() != (lens.size() - 2))
+            MIGRAPHX_THROW("inconsistent padding size");
+        auto result    = tune_attribute(val.at("padding"));
+        val["padding"] = result;
+        op.from_value(val);
+        return true;
```

Comment:
This shouldn't early return.

Reply:
I will probably rewrite the other conditional statement that checked for `normalize_axes`, which the operators modified don't have.